### PR TITLE
Endpoint routes

### PR DESCRIPTION
**Pull request by StarkRobert**
EndpointRoutes api/html added

### CODE REVIEW

File: packages/RatPack.package/RPApplication.class/instance/getAllModels.st [hunk 1]
```diff
@@ -0,0 +1,11 @@
+object handling
+getAllModels
+
+	| aJsonString |
+	aJsonString := '{"models":['.
+      self modelClasses asOrderedCollection withIndexDo: [:each :index |
```
Comment (ratnim) [inline packages/RatPack.package/RPApplication.class/instance/getAllModels.st]: Dont use with index. 
Use  collection collect: [ ....  ]   joinedSeperatedBy ','

File: packages/RatPack.package/RPApplication.class/instance/disableLogging.st [hunk 2]
```diff
@@ -1,4 +1,4 @@
 logging
 disableLogging 
 
-	self logWindow disableLogging 
\ No newline at end of file
+	self logWindow disableLogging.
```
Comment (PKuhn) [inline packages/RatPack.package/RPApplication.class/instance/disableLogging.st]: Seperator not terminator

File: packages/RatPack.package/RPApplication.class/instance/getRegisteredModelsFromRequestAsHtml..st [hunk 3]
```diff
@@ -0,0 +1,14 @@
+model handling
+getRegisteredModelsFromRequestAsHtml: aRPModelname
+ 
+      | aRPModelClass aHtmlString subStrings |
+	aRPModelClass := self getRegisteredModel: aRPModelname.
+	aHtmlString :=  '<b>Model:</b> ', aRPModelname, '<br><b>Metadata:</b> {"dataTypes":['.
```
Comment (ratnim) [inline packages/RatPack.package/RPApplication.class/instance/getRegisteredModelsFromRequestAsHtml..st]: use:

datatypes asJsonString

File: packages/RatPack.package/RPApplication.class/instance/getAllModelsFromRequestAsHtml.st [hunk 4]
```diff
@@ -0,0 +1,6 @@
+object handling
+getAllModelsFromRequestAsHtml
+	
+	 ^ self sendHtml: ('<!DOCTYPE html><html><head></head><body><h3>Models:</h3>', 
+			((self modelClasses asOrderedCollection collect: [:each | '<a href="/html/', (each name), '">', (each name), '</a><br>']) join),
```
Comment (PKuhn) [inline packages/RatPack.package/RPApplication.class/instance/getAllModelsFromRequestAsHtml.st]: Zeilenumbruch

File: packages/RatPack.package/RPApplication.class/instance/initializeLogging.st [hunk 5]
```diff
@@ -1,5 +1,5 @@
 logging
-initializeLogging  
-	
+initializeLogging
+
 	logWindow := RPLogWindow new.
-	self disableLogging
\ No newline at end of file
+	self disableLogging.
```
Comment (PKuhn) [inline packages/RatPack.package/RPApplication.class/instance/initializeLogging.st]: seperator not terminator

File: packages/RatPack.package/RPLogWindow.class/instance/disableLogging.st [hunk 6]
```diff
@@ -1,5 +1,5 @@
 as yet unclassified
 disableLogging 
 
-	loggingIsEnabled := false.
-	self closeAllViews
\ No newline at end of file
+	self closeAllViews.
+	loggingIsEnabled := false..
```
Comment (PKuhn) [inline packages/RatPack.package/RPLogWindow.class/instance/disableLogging.st]: .

File: packages/RatPack.package/RPModel.class/class/structureAsJson.st [hunk 7]
```diff
@@ -0,0 +1,14 @@
+meta datas
+structureAsJson
+	"returns datatypes and modelname as JsonString"
+
+	| aJsonString subString |
+	subString := ''.
+	aJsonString := '{"', (self name), '":{"dataTypes":['.
+	self dataTypes associations withIndexDo: [:each :index |
```
Comment (PKuhn) [inline packages/RatPack.package/RPModel.class/class/structureAsJson.st]: do not use with index

Comment (ratnim) [inline packages/RatPack.package/RPModel.class/class/structureAsJson.st]: use self datatypes associations collect: [ ... ] joinSeperatedBy: ','

**Comment by ratnim** [comment]
applay changes

**Pull request merged**